HabitTracker.BatataDoc3/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using HabitTracker.BatataDoc3.db;
+using HabitTracker.BatataDoc3.HabitTracker;
+
+namespace HabitTracker.BatataDoc3
```

Comment:
Remember that you can use file-scoped namespaces to save an indentation level:

❌ 
```
// Block-Scoped Namespace
namespace YourNamespace
{
    // ...
}
```

✔️ 
```
File-Scoped Namespace
namespace YourNamespace;

// ...
```